### DISCUSSION TITLE
Neper -V postprocessing: how to get stress-strain values

### DISCUSSION THREAD

**Discussion by zengyan0112** [neperfepx/neper#715] [Q&A] [answered]
Hi,
I was tring to anlyze the relationship beteween the stress and strain, and I want to know  how to get the stress- strain values.

**Comment by rquey** [comment, marked as answer]
Assuming you are using the `-S` module and the elemental stresses and strains are contained in your simulation directory, you can get the macroscopic values using `-resmesh stress,strain`.  This will create a file for each step under the `results/mesh/stress` and `results/mesh/strain` directories.

**Reply by zengyan0112** [reply]
Thank you very much ! 
In addition， I want to konw how to get the stress-strain curveof a specific node?

**Reply by rquey** [reply]
The stress and strain are defined at elements, not nodes. For elements, simply use the values in `results/elts`.
Please refer to the doc: https://neper.info/doc/fileformat.html#simulation-directory-sim.

**Reply by zengyan0112** [reply]
Thank you!
